src/pages/blogPost/BlogPost.jsx

```diff
@@ -0,0 +1,62 @@
+import "./BlogPost.css";
+import {useParams} from "react-router-dom";
+import dateFormatter from "../../helpers/dateFormatter.js";
+import { CaretLeft, X } from "@phosphor-icons/react";
+import {useEffect, useState} from "react";
+import axios from "axios";
+
+function BlogPost() {
+    const [postInfo, setPostInfo] = useState({});
+    const [errorMessage, setErrorMessage] = useState("");
+    const [deleteMessage, setDeleteMessage] = useState("");
+    const { postId } = useParams();
+
+    useEffect(() => {
+        async function fetchPost() {
+            try {
+                const result = await axios.get(`http://localhost:3000/posts/${postId}`);
+                setPostInfo(result.data);
+            } catch(e) {
+                console.error(e);
+                setErrorMessage("Het laden van de post is mislukt. Probeer het opnieuw.")
+            }
+        }
+        fetchPost();
+    }, []);
+
+    async function deletePost() {
+        try {
+            const result = await axios.delete(`http://localhost:3000/posts/${postId}`);
+            console.log(result);
+            setDeleteMessage("Post succesvol verwijderd!");
+        } catch (e) {
+            console.error(e);
+        }
+    }
+
+    return (
+        <div className="page-container">
+            {deleteMessage.length === 0 && Object.keys(postInfo).length === 0 && <div className="inner-container">
+                <h2 className="message">{errorMessage}</h2>
+            </div>}
+            {deleteMessage.length === 0 && Object.keys(postInfo).length > 0 && <div className="inner-container">
```

Comment:
Dan kan hier de check wat makkelijk
```suggestion
            {deleteMessage.length === 0 && postInfo && <div className="inner-container">
                <h2 className="message">{errorMessage}</h2>
            </div>}
            {deleteMessage.length === 0 && postInfo && <div className="inner-container">
```